lib/core/src/sdk.rs

```diff
@@ -3304,13 +3301,14 @@ impl LiquidSdk {
     }
 
     /// Empties the Liquid Wallet cache for the [Config::network].
+    #[cfg(not(all(target_family = "wasm", target_os = "unknown")))]
```

Comment:
Does this need removing from the wasm crate interface also?

Reply:
Nice catch! Removed in [bbde082](https://github.com/breez/breez-sdk-liquid/pull/792/commits/bbde082f304a4da3034749c6ab1091011992e0e1).